cmake/unifex_env.cmake

```diff
@@ -19,7 +19,7 @@ else()
   message(WARNING "[unifex warning]: unknown compiler ${CMAKE_CXX_COMPILER_ID} !")
 endif()
 
-if (MSVC)
```

Comment:
Not sure why the cmake variable here was called `MSVC`. I assume it was a mistake? Or an incomplete edit during maintenance? @ispeters ?